www/js/lib/zimArchive.js

```diff
@@ -237,7 +238,7 @@ define(['zimfile', 'zimDirEntry', 'util', 'utf8'],
                 return that._file.dirEntryByTitleIndex(index).then(function(dirEntry) {
                     var title = dirEntry.getTitleOrUrl();
                     // Only return dirEntries with titles that actually begin with prefix
-                    if (dirEntry.namespace === 'A' && title.indexOf(prefix) === 0) {
+                    if (/[AC]/.test(dirEntry.namespace) && title.indexOf(prefix) === 0) {
```

Comment:
Should the namespace test be replaced with a MIME type test? Probably not, becuase it involves a lookup and for old-style ZIMs the `A` namespace test is sufficient and much faster. And for new-style ZIMs, we should use the v2 `titlePtrList`, which only contains articles, so a test is redundant.